backend/api/parse_pdf.py

```
@@ -520,6 +520,40 @@ async def get_default_personas():
         ]
     }
 
+async def create_pdf_metadata(main_file_data: dict, session_id: Optional[str] = None) -> dict:
```

Ruff (0.14.3)

Unused function argument: `session_id`

(ARG001)